internal/db/db_blueprints.go

```diff
@@ -68,12 +86,46 @@ func (db *dB) GetBlueprint(id uuid.UUID, orgID, accountNumber string) (*Blueprin
 	row := conn.QueryRow(ctx, sqlGetBlueprint, id, orgID, accountNumber)
 	err = row.Scan(&result.Id, &result.VersionId, &result.Name, &result.Description, &result.Version, &result.Body)
 	if err != nil {
-		return nil, err
+		if errors.Is(err, pgx.ErrNoRows) {
+			return nil, BlueprintNotFoundError
+		} else {
+			return nil, err
+		}
 	}
 
 	return &result, err
 }
 
+func (db *dB) UpdateBlueprint(id uuid.UUID, blueprintId uuid.UUID, orgId string, name string, description string, body json.RawMessage) error {
+	ctx := context.Background()
+	conn, err := db.Pool.Acquire(ctx)
+	if err != nil {
+		return err
+	}
+	defer conn.Release()
+
+	err = db.withTransaction(ctx, func(tx pgx.Tx) error {
+		tag, txErr := tx.Exec(ctx, sqlUpdateBlueprint, blueprintId, orgId, name, description)
+		if tag.RowsAffected() != 1 {
+			return fmt.Errorf("%w: %w, expected 1, returned %d", BlueprintNotFoundError, AffectedRowsMismatchError, tag.RowsAffected())
```

Comment:
linter does not like this and neighter do I TBH xD

```
internal/db/db_blueprints.go:110: fmt.Errorf call has more than one error-wrapping directive %w
```

I think just not using constants here would be good enough.

```suggestion
			return fmt.Errorf("blueprint not found: %w, expected 1, returned %d", AffectedRowsMismatchError, tag.RowsAffected())
```

The only use for the constant would be to programmatically distinguish wether the issue was with the first or the second query. But I don't see us needing it ATM :)

Reply:
Yeah this is only possible in Go 1.21+ but this project is still on 1.19.